qiskit_nature/drivers/second_quantization/pyscfd/pyscfdriver.py

```diff
@@ -10,25 +10,39 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-""" PYSCF Driver """
+"""The PySCF Driver."""
 
 import importlib
 import logging
+import os
+import tempfile
+import warnings
 from enum import Enum
-from typing import Optional, Union, List
+from typing import List, Optional, Tuple, Union
 
+import numpy as np
 from qiskit.utils.validation import validate_min
 
-from qiskit_nature import QiskitNatureError
-
+from ....exceptions import QiskitNatureError
+from ..fermionic_driver import FermionicDriver, MethodType
 from ..qmolecule import QMolecule
-from .integrals import compute_integrals
-from ..fermionic_driver import FermionicDriver, HFMethodType
 from ...molecule import Molecule
 from ...units_type import UnitsType
 
 logger = logging.getLogger(__name__)
 
+try:
+    from pyscf import __version__ as pyscf_version
+    from pyscf import ao2mo, dft, gto, scf
+    from pyscf.lib import chkfile as lib_chkfile
+    from pyscf.lib import logger as pylogger
+    from pyscf.lib import param
+    from pyscf.tools import dump_mat
+
+    warnings.filterwarnings("ignore", category=DeprecationWarning, module="pyscf")
+except ImportError:
+    logger.info("PySCF is not installed. See https://sunqm.github.io/pyscf/install.html")
```

Comment:
The former code was fixed with an updated message for the changed install location as follows
```suggestion
    logger.info("PySCF is not installed. See https://pyscf.org/install.html")
```

Reply:
Thanks! I must have missed this one. After fixing it grepping for `sunqm` no longer returns any results in the codebase :+1: